src/lib/messages/sensor/atlas_scientific__oem_ec.proto

```diff
@@ -11,6 +11,7 @@ message AtlasScientificOEMEC
     };
 
     optional double conductivity = 1;
+    optional double specific_conductivity = 2;
```

Comment:
It looks like this field doesn't get used (yet).  Should we do the `specific_conductivity` and `salinity_calculated` in the fusion app (for example), so those derived calculations are only done in one place?

Reply:
I see what you mean (Using the DRY method). It really comes down to the report speed. The Hydros are eventually going to start using the stm32 board which would deprecate the python drivers for the pressure and conductivity sensors.